samples/dotnet/kernel-syntax-examples/Resources/EmbeddedResource.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.IO;
+using System.Reflection;
+using RepoUtils;
+
+namespace Resources;
+
+internal static class EmbeddedResource
+{
+    private static readonly string? s_namespace = typeof(EmbeddedResource).Namespace;
+
+    internal static string Read(string name)
+    {
+        var assembly = typeof(EmbeddedResource).GetTypeInfo().Assembly;
+        if (assembly == null) { throw new ConfigurationException($"[{s_namespace}] {name} assembly not found"); }
```

Comment:
Is this even a possible situation??

Reply:
see other comment, also the signature of `.Assembly`. .NET allows that to be null, so I'm making sure it's handled